doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1838,6 +1930,7 @@ following configuration:
   - members' sign keys and certificates (that are stored in soft token)
   - security server's internal TLS key and certificate
   - security server's UI key and certificate
+  - messagelog signing and encryption keys
```

Comment:
Should this be:

- messagelog database encryption keys
- messagelog archives signing and encryption keys

Reply:
@jhyoty Could you confirm is the above comment is correct?

Reply:
It is not well defined. If the archive encryption and signing keys are in /etc/xroad/gpghome (or somewhere outside of /etc/xroad), those are not included in the backup. Similarly for the database encryption keys (there is no default location for the keystore).

Reply:
In that case we need to mention it in the documentation. I'd remove "messagelog signing and encryption keys" from the list of items that are included in the backups and add the following note right after the list.

> **N.B.** Message log database encryption keys, and message log archives encryption and signing keys are included in the backups only if they are stored under the `/etc/xroad` directory. However, they should not be stored in the `/etc/xroad/gpghome` subdirectory since it is excluded from the backups.

Also, I think we should mention the backup aspect elsewhere in the documentation too where the properties that are used to defined the locations of the keys are explained. Otherwise there's a big risk that the user assumes they're included in the backups even if they're not. Maybe this could be shown in the diagnostics view as well - message log archives encryption and sign keys included in the backups, message log encryption keys included in the backups.

Reply:
Clarified in https://github.com/nordic-institute/X-Road/pull/1038/commits/857c91804c605c87114ea3013b6620fb09045d7a